ios-base/DroidKaigi 2020/Session/Views/Cells/SessionCell.swift

```diff
@@ -97,7 +96,7 @@ final class SessionCell: UICollectionViewCell {
                         ImageCache.shared[request] = response.image // Set cache
                         speakerIconView.image = ImageCache.shared[request]
                     case .failure:
-                        speakerIconView.image = ImageLoadingOptions.shared.failureImage
+                        speakerIconView.backgroundColor = UIColor.white
```

Comment:
[In the previous commit](https://github.com/DroidKaigi/conference-app-2020/pull/807/commits/c8f24931724b74dc5ef153b22b130238bd6b4d3a#diff-e9565e55cb987a855b953b2f45983c23R84), failure image was set to logo, but I changed it because I thought the white background was appropriate.